test/_init_tests.js

```diff
@@ -42,6 +42,8 @@ import PreactEnzyme from 'enzyme-adapter-preact-pure';
 import sinon from /*OK*/ 'sinon';
 import stringify from 'json-stable-stringify';
 
+chai.use(require('chai-as-promised')); // eslint-disable-line
```

Comment:
Does it make sense to move this to the bottom where the custom chai assertions are made or keep it near the imports? Or maybe could we move all the chai stuff out of `_init_tests.js` because it's such a long file?

Reply:
I opted for moving it next to where the chai assertions are made. If we want to move all the chai code elsewhere, I'd prefer to do it in a separate PR

Reply:
Sg!